plugins/web/opentelemetry-instrumentation-long-task/src/instrumentation.ts

```diff
@@ -41,13 +44,18 @@ export class LongTaskInstrumentation extends InstrumentationBase {
   moduleName = this.component;
 
   private _observer?: PerformanceObserver;
+  private _observerCallback?: ObserverCallback;
 
   /**
    *
    * @param config
    */
-  constructor(config: InstrumentationConfig = {}) {
+  constructor(
+    config: InstrumentationConfig = {},
+    observerCallback?: ObserverCallback
```

Comment:
I am not very familiar with the web instrumentations, but in node we usually add this kind of hook in the `config` object.
Does it make sense to do that for web instrumentations as well?

Reply:
Totally makes sense.